bank/bank-backend/dbos-test-config.yaml

```
@@ -0,0 +1,13 @@
+database:
+  hostname: 'localhost'
+  port: 5432
+  username: 'bank_a'
+  connectionTimeoutMillis: 3000
+  app_db_name: 'bank_a'
+  password: ${PGPASSWORD}
+  app_db_client: "prisma"
+runtimeConfig:
+  port: 8081
+application:
+  bankname: "testbank"
+  bankport: '8091'
```

We had a typo here for a while (my bad) :)
```suggestion
  bankport: '8081'
```